source/manual/alerts/signon-api-user-token-expiry-soon.md

```diff
@@ -0,0 +1,68 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Alertmanager SignonApiUserTokenExpirySoon
+parent: "/manual.html"
+layout: manual_layout
+section: Alertmanager alerts
+---
+
+[signon]: https://signon.publishing.service.gov.uk/api_users
+[gds-api-adapters]: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api.rb
+
+One or more tokens for API Users are about to expire. You should rotate
+expiring tokens to ensure the associated application keeps working.
+
+> If the tokens are
+> `New London: Trade Tariff Admin (PaaS) token for New London: Trade Tariff Backend (PaaS)`
+> see [Trade Tariff Admin on the Wiki](https://gov-uk.atlassian.net/wiki/spaces/PLOPS/pages/3155099649/Trade+Tariff+Admin)
+>
+> If the tokens are for the `Signon API Client (permission and suspension updater)`
+> you should contact the permissions team.
```

Comment:
For readability, I'd move this lower in the docs - as its very unlikely the alert will be about these apps.